jinja2-mode.el

```diff
@@ -322,6 +322,7 @@
 
 ;;;###autoload
 (add-to-list 'auto-mode-alist '("\\.jinja2\\'" . jinja2-mode))
+(add-to-list 'auto-mode-alist '("\\.j2\\'" . jinja2-mode))
```

Comment:
You need to add `;;;###autoload` above this line as well, or wrap both together in a `progn`.

Reply:
Thanks. I updated my PR. @paradoxxxzero can you merge this in?